app/src/main/res/menu/download_log.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:custom="http://schemas.android.com/apk/res-auto">
+
+    <item
+        android:id="@+id/refresh_item"
+        android:title="@string/refresh_label"
```

Comment:
Title and id should be something like `retry_all`, not refresh. This could lead to confusion, otherwise.